Jenkinsfile

```diff
@@ -76,3 +95,15 @@ void withMavenEnv(List envVars = [], def body) {
         body.call()
     }
 }
+
+void renameFiles(def files, String prefix) {
+    for(i = 0; i < files.length; i++) {
```

Comment:
I don't think I can use File because the builds are running on nodes, but the pipeline is running on the master. That is my understanding at least. I would also think that file operations would not be whitelisted by default.